tests/spec_tests/config.yaml

```diff
@@ -24,9 +24,61 @@ wdl-1.1:
     - test_sub.wdl                # issue #709
     - map_to_struct.wdl           # issue #712
   skip:
-    # these have heavyweight dependencies
+    - write_json_fail.wdl # FIXME: refuse to string-coerce map keys
+    - test_gpu_task.wdl   # hardware-dependent
+    # heavyweight dependencies:
     - hisat2_task.wdl
     - gatk_haplotype_caller_task.wdl
 wdl-1.2:
-  xfail: []
-  skip: []
+  xfail:
+    - relative_and_absolute_task.wdl  # issue #214
+    - test_struct.wdl                 # issue #728
+    # To be implemented:
+    - allow_nested.wdl
+    - chunk_array.wdl
+    - file_sizes_task.wdl
+    - multi_nested_inputs.wdl
+    - import_structs.wdl
+    - input_hint_task.wdl
+    - join_paths_task.wdl
+    - one_mount_point_task.wdl
+    - person_struct_task.wdl
+    - string_to_file.wdl
+    - test_allow_nested_inputs.wdl
+    - test_contains.wdl
+    - test_find_task.wdl
+    - test_hints_task.wdl
+    - test_keys.wdl
+    - test_matches_task.wdl
+    - test_runtime_info_task.wdl
+    - test_select_first.wdl
+    # other spec bugs:
+    - all_return_codes_task.wdl          # issue #729
+    - hello_parallel.wdl                 # expected output is wrong
+    - map_to_struct.wdl                  # issue #712
+    - multiline_string_placeholders.wdl  # wrong namespace in expected output
+    - nested_scatter.wdl                 # expected output is wrong
+    - placeholders.wdl                   # wrong namespace in expected output
+    - placeholder_none.wdl               # questionable claim that interpolations should swallow errors
+    - primitive_literals.wdl             # expected output is wrong
+    - serde_pair.wdl                     # expected output is wrong
+    - single_return_code_task.wdl        # issue #729
+    - struct_to_struct.wdl               # typo capitalized Struct
+    - test_contains_key.wdl              # uses python-style `a if c else b` instead of `if c then a else b
+    - test_length.wdl                    # typo in map literal
+    - test_sub.wdl                       # issue #709
+    - test_values.wdl                    # str_to_files should be str_to_ints
+    - write_tsv_task.wdl                 # missing Numbers struct definition
```

Comment:
@vsmalladi I made a pass through the spec examples in wdl-1.2.1 and found a bunch of errata listed here. Would I be right to guess a lot of them are the subject of your https://github.com/openwdl/wdl/pull/702? BTW, I've found that o3 is quite good at debugging these!

Reply:
@mlin and @claymcleod can you review that pr so that we can merge these issues.